src/app/issuer/components/issuer-create/issuer-create.component.html

```diff
@@ -1,12 +1,11 @@
-<form-message></form-message>
-<div class="topbar">
-	<div class="l-containerxaxis">
-		<div class="topbar-x-heading">{{ 'Issuer.createInstitution' | translate }}</div>
-		<div class="topbar-x-subheading">{{ 'Issuer.awardBadeges' | translate }}</div>
+<div class="oeb page-padding">
```

Comment:
I think we need to wrap this once more to avoid safari issues:
add a div around this with "tw-relative tw-w-full"

Reply:
done.